javascript/javascript1/week4/inClassExercises.js

```diff
@@ -0,0 +1,138 @@
+////////////////////////////////////////////////////////////////////Fizz Buzz
+
+
+function number(x1, x2) {
+    let num = x1;
+    console.log(num);
+    for (let i = x1; i < x2; i++) {
+        num += 1;
+        if ((num % 5 === 0) && (num % 3 === 0)) {
+            console.log('FizzBuzz');
+        } else if (num % 5 === 0) {
+            console.log('Buzz');
+        } else if (num % 3 === 0) {
+            console.log('Fizz');
+        } else {
+            console.log(num);
+        }
+    }
+}
+number(4, 12);
```

Comment:
Always try to name the function as well as possible. `number` is not a good name for a function. 

Also, no need to initialize `num = x1` when you're already iterating over all the numbers between `x1` and `x2` in the `for` loop. This code would  do the same: 

```
function fizzBuzz(x1, x2) {

    for (let num = x1; num < x2; num++) {
        if (!(num % 5) && !(num % 3)) {
            console.log(`${num}: FizzBuzz`);

        } else if (!(num % 5)) {
            console.log(`${num}: Buzz`);

        } else if (!(num % 3)) {
            console.log(`${num}: Fizz`);

        } else {
            console.log(num);
        }
    }
} 
```
`!` is the logical NOT and `!(num%3)` is the same as `num%3 === 0`.